YoutubeExplode.Tests/TestData/PlaylistIds.cs

```diff
@@ -11,4 +11,5 @@ internal static class PlaylistIds
     public const string UserUploads = "UUTMt7iMWa7jy0fNXIktwyLA";
     public const string Weird = "PL601B2E69B03FAB9D";
     public const string ContainsLongVideos = "PLkk2FsMngwGi9FNkWIoNZlfqglcldj_Zs";
+    public const string MixPlaylist = "RDTsYhxMnGYCw";
```

Comment:
Can you please rename this to `VideoMix` and move it above `MusicMix`? Just to keep things consistent. All playlists starting with `RD` are mix playlists, but they behave a bit differently.